sszgen/unmarshal.go

```diff
@@ -268,26 +274,29 @@ func (v *Value) umarshalContainer(start bool, dst string) (str string) {
 			if {{.offset}} = ssz.ReadOffset({{.dst}}); {{.offset}} > size {{.more}} {
 				return ssz.ErrOffset
 			}
+			if {{.offset}} < {{.offsetMinimum}} {
```

Comment:
You can reuse the "more" variable to include the first offset check. Something like this should work:
```
if prev, ok := offsetsMatch[offset]; ok {
    // check that the offsets are higher than the previous one
    data["more"] = fmt.Sprintf(" || %s > %s", prev, offset)
} else {
   // check that the first offset starts at the correct position
   data["more"] = fmt.Sprintf(" || %s > %d", offset, v.n)
}
```